registration-processor-credential-partners.json

```diff
@@ -25,6 +25,15 @@
       "template": "RPR_UIN_CARD_TEMPLATE",
       "process": ["OPENCRVS_NEW"],
       "metaInfoFields": ["opencrvsBRN"]
+    },
+    {
+      "id": "digitalCardInsurancePartner",
+      "partnerId": "mpartner-default-digitalinsuarance2",
+      "credentialType": "vercred",
+      "template": "RPR_UIN_CARD_TEMPLATE",
+      "appIdBasedCredentialIdSuffix": null,
+      "process": null,
+      "metaInfoFields": null
```

Comment:
Is this required to be part of default config ?

Reply:
confirmed with @dhanendra06. yes, it is required.

Reply:
@dhanendra06 to remove this as discussed

Reply:
Done